### PR TITLE
issue 022 : added duration days env variable

### PR DESCRIPTION
**Pull request by sandragolden**
resolves issue #22 : added duration days as an .env variable to make the number of days a scratchOrg will live configurable prior to deployment.

Please let me know if anything was mixed or needs to be adjusted.

### CODE REVIEW

File: lib/cli-api/_common/_getScratchOrgDurationDays.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    // Maintain the specified duration days value -- provided that it validates
+    if (validatorOutput.validationResult === true) { output = scratchOrgDurationDays; }
+
+    // Return the alias to leverage
+    return output;
+
+}
+
+// Export the function
+module.exports = getScratchOrgDurationDays;
```
Comment (jbachelet) [inline lib/cli-api/_common/_getScratchOrgDurationDays.js:37]: I would suggest the following version of the function:

```suggestion
module.exports = scratchOrgDurationDays => validators.validateDurationDays(scratchOrgDurationDays).validationResult === true ? scratchOrgDurationDays : config.get('sfScratchOrg.durationDays');
```

**Review by jbachelet — CHANGES_REQUESTED** [review]

File: lib/cli-api/_common/_getScratchOrgDurationDays.js (hunk 1 above)
Comment (sandragolden) [reply lib/cli-api/_common/_getScratchOrgDurationDays.js]: updated, much cleaner, thanks!

Comment (b2csolutionarchitect) [reply lib/cli-api/_common/_getScratchOrgDurationDays.js]: Thank you @sandragolden and @jbachelet!  You're making history. :)

**Review by b2csolutionarchitect — APPROVED** [review]
... nice work @sandragolden; real curious to get your feedback re: extending the CLI framework we're leveraging.  Thank you for your efforts!

File: lib/cli-api/_common/_getScratchOrgDurationDays.js (hunk 1 above)
Comment (b2csolutionarchitect) [reply lib/cli-api/_common/_getScratchOrgDurationDays.js]: ... and by the way, @jbachelet did the EXACT SAME THING to me.

**Pull request merged**